Gemfile

```diff
@@ -24,7 +24,7 @@ gem 'jbuilder', '~> 2.7'
 
 # Use Active Storage variant
 # gem 'image_processing', '~> 1.2'
-
+gem 'bcrypt', '3.1.13'
```

Comment:
gem này để làm gì á e

Reply:
dạ gem bcrypt để mã hóa mật khẩu người dùng một cách an toàn.